tests/components/universal/test_media_player.py

```diff
@@ -681,6 +681,187 @@ def test_supported_features_children_and_cmds(self):
+        assert len(service) == 8
+        self.hass.services.call(
+            "media_player",
+            "repeat_set",
+            service_data={"entity_id": "media_player.test"},
+            blocking=True,
+        )
+        assert len(service) == 9
+        self.hass.services.call(
+            "media_player",
+            "shuffle_set",
+            service_data={"entity_id": "media_player.test"},
+            blocking=True,
+        )
+        assert len(service) == 10
+        self.hass.services.call(
+            "media_player",
+            "media_play",
+            service_data={"entity_id": "media_player.test"},
+            blocking=True,
+        )
+        assert len(service) == 11
+        self.hass.services.call(
+            "media_player",
+            "media_pause",
+            service_data={"entity_id": "media_player.test"},
+            blocking=True,
+        )
+        assert len(service) == 12
+        self.hass.services.call(
+            "media_player",
+            "media_stop",
+            service_data={"entity_id": "media_player.test"},
+            blocking=True,
+        )
+        assert len(service) == 13
+        self.hass.services.call(
+            "media_player",
+            "media_next_track",
+            service_data={"entity_id": "media_player.test"},
+            blocking=True,
+        )
+        assert len(service) == 14
+        self.hass.services.call(
+            "media_player",
+            "media_previous_track",
+            service_data={"entity_id": "media_player.test"},
+            blocking=True,
+        )
+        assert len(service) == 15
+        self.hass.services.call(
+            "media_player",
+            "toggle",
+            service_data={"entity_id": "media_player.test"},
+            blocking=True,
+        )
+        assert len(service) == 16
+        self.hass.services.call(
+            "media_player",
+            "clear_playlist",
+            service_data={"entity_id": "media_player.test"},
+            blocking=True,
+        )
+        assert len(service) == 17
+        self.hass.services.call(
+            "media_player",
+            "media_play_pause",
+            service_data={"entity_id": "media_player.test"},
+            blocking=True,
+        )
+        assert len(service) == 18
+        self.hass.services.call(
+            "media_player",
+            "play_media",
+            service_data={"entity_id": "media_player.test"},
+            blocking=True,
+        )
+        assert len(service) == 19
+        self.hass.services.call(
+            "media_player",
+            "media_seek",
+            service_data={"entity_id": "media_player.test"},
+            blocking=True,
+        )
+        assert len(service) == 20
+        self.hass.services.call(
+            "media_player",
+            "toggle",
+            service_data={"entity_id": "media_player.test"},
+            blocking=True,
+        )
+        assert len(service) == 21
```

Comment:
Martin, a little question: is it allowed to do this routine in loops?

Reply:
Yes, if we can allow any conditional checks.